graphitesend/block_metric.py

```diff
@@ -1,4 +1,8 @@
-from contextlib2 import contextmanager, ExitStack
+try:
+  from contextlib import contextmanager, ExitStack
+except ImportError:
+  from contextlib2 import contextmanager, ExitStack
+
```

Comment:
Let's use `sys.version_info.major` here also and move that logic after the other imports.

Reply:
ok let me transform that